lib/repo_calculator.rb

```diff
@@ -0,0 +1,22 @@
+require_relative "data_retriever"
+require "pry"
+
+class RepoCalculator
+  attr_reader :points, :hash
+
+  def initialize(name)
+    @points = 0
+    @hash = DataRetriever.new(name, {url: "/repos", filename: "#{name}_reops"}).get_data
```

Comment:
Line is too long. [88/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.